frontend/app-preview/src/components/PreviewControlHeader/PreviewControlHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { type ReactElement, type ChangeEvent } from 'react';
+import classes from './PreviewControlHeader.module.css';
+import { useTranslation } from 'react-i18next';
+import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { StudioNativeSelect } from '@studio/components';
+import { ToggleGroup } from '@digdir/designsystemet-react';
+
+export type PreviewControlHeaderProps = {
+  viewSize: 'desktop' | 'mobile';
+  setViewSize: (value: any) => void;
+  selectedLayoutSet: string | null;
+  handleChangeLayoutSet: (value: string) => void;
+};
+
+export const PreviewControlHeader = ({
+  viewSize,
+  setViewSize,
+  selectedLayoutSet,
+  handleChangeLayoutSet,
+}: PreviewControlHeaderProps): ReactElement => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { data: layoutSets } = useLayoutSetsQuery(org, app);
```

Comment:
Should we pass the layoutSets as a prop instead of using useQuery here? I think that might simplify the tests as well. 🤔 If the header is responsible for data fetching, we should also add a loading state. 😊 

```suggestion
export const PreviewControlHeader = ({
  viewSize,
  setViewSize,
  selectedLayoutSet,
  handleChangeLayoutSet,
  layoutSets
}: PreviewControlHeaderProps): ReactElement => {
  const { t } = useTranslation();
  const { org, app } = useStudioEnvironmentParams();
```

Reply:
I think it is better to have the query in this component as it is not used in or related to the code in the parent component. I've added a loading spinner in this component 😄